frontend/src/environments/environment.ts

```diff
@@ -3,6 +3,7 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
+  apiUrl: 'https://localhost:5001',
```

Comment:
```suggestion
  apiUrl: 'https://localhost:5001/api',
```

I think we'll have it